docker/prognostic_run/Dockerfile

```diff
@@ -30,36 +50,35 @@ COPY workflows/post_process_run /fv3net/workflows/post_process_run
 COPY workflows/prognostic_c48_run/ /fv3net/workflows/prognostic_c48_run
 COPY external/emulation /fv3net/external/emulation
 COPY external/radiation /fv3net/external/radiation
-RUN bash install_fv3net_packages.sh \
-    /fv3net/external/vcm \
-    /fv3net/external/artifacts \
-    /fv3net/external/loaders \
-    /fv3net/external/fv3fit \
-    /fv3net/external/fv3kube \
-    /fv3net/workflows/post_process_run \
-    /fv3net/workflows/prognostic_c48_run \
-    /fv3net/external/emulation \
-    /fv3net/external/radiation
+RUN bash /fv3net/.environment-scripts/setup_environment.sh \
+    fv3net-packages \
+    ${PLATFORM} \
+    ${CLONE_PREFIX} \
+    ${INSTALL_PREFIX} \
+    ${FV3NET_DIR} \
+    ${CALLPYFORT}
+
+# Add emulation project scripts
+COPY projects/microphysics/scripts /fv3net/projects/microphysics/scripts
 
 RUN echo "ulimit -s unlimited" >> /etc/bash.bashrc && \
     mkdir /outdir && \
     chmod -R 777 /outdir
 
-
 # these are needed for "click" to work
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
 # Override microphysics emulation
-ENV VAR_META_PATH=/fv3net/external/emulation/microphysics_parameter_metadata.yaml
+ENV VAR_META_PATH=$FV3NET_DIR/external/emulation/microphysics_parameter_metadata.yaml
 ENV OUTPUT_FREQ_SEC=18000
 
 # Add emulation project scripts
-COPY projects/microphysics/scripts /fv3net/projects/microphysics/scripts
-RUN chmod +x /fv3net/projects/microphysics/scripts/*
-ENV PATH=/fv3net/projects/microphysics/scripts:${PATH}
+ENV PATH=$FV3NET_DIR/projects/microphysics/scripts:${PATH}
+
+# Add fv3net packages to the PYTHONPATH
+ENV PYTHONPATH=$FV3NET_DIR/workflows/prognostic_c48_run:$FV3NET_DIR/external/fv3fit:$FV3NET_DIR/external/emulation:$FV3NET_DIR/external/vcm:/fv3net/external/artifacts:$FV3NET_DIR/external/loaders:$FV3NET_DIR/external/fv3kube:$FV3NET_DIR/workflows/post_process_run:$FV3NET_DIR/external/radiation:${PYTHONPATH}
```

Comment:
did you previously have this as a call to `activate_environment.sh`? What's the reason for not doing that?

Reply:
Indeed, I reverted it in 736c6539d33c95f183ed8806dc2e24311b0a6ff8.  I didn't have a strong reason other than it felt convoluted to handle through an entrypoint (it's awkward that we can't simply source a script to set some environment variables in the Docker image).  I can bring that back if you don't think it's so bad.  It does have the benefit that it defines all of those variables in one place (but it is a bit of indirection in the Dockerfile).

Reply:
Got it. Yeah I agree using an entrypoint is a bit overkill. I think it's fine as is.